internal/api/block_documents.go

```diff
@@ -8,6 +8,7 @@ import (
 
 type BlockDocumentClient interface {
 	Get(ctx context.Context, id uuid.UUID) (*BlockDocument, error)
+	GetByName(ctx context.Context, typeSlug, name string) (*BlockDocument, error)
```

Comment:
tiny nit - `GetByTypeSlug`

Reply:
nvm, i misread this

Reply:
Could do `GetBySlugAndName` but felt ugly 😆